src/unix/notbsd/android/mod.rs

```diff
@@ -562,6 +565,7 @@ pub const TIOCMBIC: ::c_int = 0x5417;
 pub const TIOCMSET: ::c_int = 0x5418;
 pub const FIONREAD: ::c_int = 0x541B;
 pub const TIOCCONS: ::c_int = 0x541D;
+pub const CLONE_NEWCGROUP: ::c_int = 0x02000000;
```

Comment:
I promise this is the last nitpick :P 

Maybe this should be moved along with https://github.com/rust-lang/libc/search?utf8=%E2%9C%93&q=CLONE_NEWCGROUP&type= to be with the others ? https://github.com/rust-lang/libc/blob/83f5543e761bad58c1f061447c28df4a0201c761/src/unix/notbsd/mod.rs#L689

It's not defined in sparc64, but I strongly believe it to be a bug since I can't find any mention of this flag not being defined on SPARC for linux.